ui/plugins/pluginpanel.html

```diff
@@ -68,33 +68,34 @@
             </a>
           </li>
           <li>
-            <!--the first 3 items are in the same box-->
-            <ul class="nav navbar-nav">
-              <li>
-                <a id="pin-job-btn" href=""
-                   title="Add this job to the pinboard"
-                   ng-click="pinboard_service.pinJob(selectedJob)">
-                  <span class="glyphicon glyphicon-pushpin"
-                        ng-class="{'icon-blue': (pinboard_service.count.numPinnedJobs > 0)}">
-                  </span>
-                  <span ng-show="pinboard_service.count.numPinnedJobs"
-                        class="pinned-job-count">{{getCountPinnedJobs()}}</span>
-                </a>
-              </li>
-
-            </ul>
+            <a id="pin-job-btn" href=""
+               title="Add this job to the pinboard"
+               ng-click="pinboard_service.pinJob(selectedJob)">
+              <span class="glyphicon glyphicon-pushpin"
+                    ng-class="{'icon-blue': (pinboard_service.count.numPinnedJobs > 0)}">
+              </span>
+              <span ng-show="pinboard_service.count.numPinnedJobs"
+                    class="pinned-job-count">{{getCountPinnedJobs()}}</span>
+            </a>
           </li>
-          <li class="dropdown">
-            <a id="retriggerLabel"
-               ng-attr-title="{{user.loggedin ? 'Retrigger or backfill this job' :
+          <li>
+            <button id="retrigger-btn" href=""
+               ng-attr-title="{{user.loggedin ? 'Repeat the selected job' :
                               'Must be logged in to retrigger a job'}}"
-               ng-class="user.loggedin ? 'icon-green' : 'disabled'" href="" data-toggle="dropdown">
-              <span class="fa fa-repeat"></span>&nbsp
-              <span class="fa fa-angle-down lightgray"></span>
-            </a>
-            <ul class="dropdown-menu actionbar-menu" role="menu" aria-labelledby="retriggerLabel"
-                ng-include src="'partials/main/thJobDetailsRetriggerMenu.html'">
-            </ul>
+               ng-class="user.loggedin ? 'icon-green' : 'disabled'"
+               ng-disabled="!user.loggedin"
+               ng-click="retriggerJob([selectedJob])">
+              <span class="fa fa-repeat"></span>
+            </button>
+          </li>
+          <li>
+            <button id="backfill-btn" href=""
+               ng-attr-title="{{backfillButtonTitle()}}"
+               ng-class="canBackfill() ? 'icon-cyan' : 'disabled'"
+               ng-disabled="!canBackfill()"
+               ng-click="backfillJob()">
+              <span class="fa fa-arrow-down"></span>
+            </button>
```

Comment:
The new changes in this block of markup are mostly, using `<button>`, the new `backfillButtonTitle()` function, and inclusion of the missing `ng-click` functions from the retired menu.